src/ImageSharp/PixelFormats/PixelImplementations/PixelOperations/RgbaVector.PixelOperations.cs

```diff
@@ -21,7 +21,7 @@ public partial struct RgbaVector
         internal class PixelOperations : PixelOperations<RgbaVector>
         {
             private static readonly Lazy<PixelTypeInfo> LazyInfo =
-                new Lazy<PixelTypeInfo>(() => PixelTypeInfo.Create<RgbaVector>(PixelAlphaRepresentation.Unassociated), true);
+                new(() => PixelTypeInfo.Create<RgbaVector>(PixelAlphaRepresentation.Unassociated), true);
```

Comment:
I have to say I am not a fan of this syntax for object creation... I find it a lot harder to read. especially in cases like this:

```c#
// in some piece of unchanged code
var list = new List<ObjType>();
.
.
.
// lots of lines later 
.
.
.
// in the middle of a PR change block, there is no context available while review what the hell this thing is.
list.Add(new(arg1, "arg2", 34)
{
    OptionalProperty = bar
});
```

not going to block any PRs etc over this just wanted to comment on the fact I feel it drastically harms readability in most cases... the only exception I can kind of agree on is property initialization where `var` is not allowed. Or possibly collection initializers where the Type of clear in the collection type.

Reply:
Yeah, definitely something to be aware of. I really like it in some scenarios though